src/pykeen/hpo/hpo.py

```diff
@@ -909,7 +909,7 @@ def suggest_discrete_power_int(trial: Trial, name: str, low: int, high: int, bas
     """Suggest an integer in the given range [2^low, 2^high]."""
     if high <= low:
         raise Exception(f"Upper bound {high} is not greater than lower bound {low}.")
-    choices = [base**i for i in range(low, high + 1)]
+    choices = [base ** i for i in range(low, high + 1)]
```

Comment:
@migalkin it seems strange that linting re-formatted files not touched by this PR 🤔

Reply:
@migalkin you can always hard reinstall everything by running `tox -r`

Reply:
@cthoyt I think I saw somewhere (although I cannot find it right now) that the black version was pinned, as the most recent version changed its default on spaces around operators(?). Could this be the reason?

Reply:
I pinned it for a moment because flake8-black broke, but that's all fixed now.